src/web/mjs/connectors/SweetTimeScan.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadaraNovel from './templates/WordPressMadaraNovel.mjs';
+
+export default class SweetTimeScan extends WordPressMadaraNovel {
+
+    constructor() {
+        super();
+        super.id = 'sweettimescan';
+        super.label = 'Sweet Time Scan';
+        this.tags = [ 'manga', 'webtoon', 'portuguese' ];
+        this.url = 'https://sweetscan.net';
+    }
```

Comment:
Is there no need to define `this.novelObstaclesQuery` to remove potential overlays from novels?

Reply:
> Is there no need to define `this.novelObstaclesQuery` to remove potential overlays from novels?

No, dont have AD